app/scenes/channel_info/channel_info_container.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+
+import {getChannelStats} from 'service/actions/channels';
+import {getCurrentChannel, getCurrentChannelStats, getChannelsByCategory} from 'service/selectors/entities/channels';
+import {getTheme} from 'service/selectors/entities/preferences';
+import {getCurrentTeamUrl} from 'service/selectors/entities/teams';
+import {getUser} from 'service/selectors/entities/users';
+
+import ChannelInfo from './channel_info';
+
+function mapStateToProps(state, ownProps) {
+    const currentChannel = getCurrentChannel(state);
+    const currentChannelCreator = getUser(state, currentChannel.creator_id);
+    const currentChannelCreatorName = currentChannelCreator && currentChannelCreator.username;
+    const currentChannelMemberCount = getCurrentChannelStats(state) && getCurrentChannelStats(state).member_count;
+    const currentChannelUrl = `${getCurrentTeamUrl(state)}/channels/${currentChannel.name}`;
```

Comment:
I'm wondering if we should show the channel URL here, @lfbrock @asaadmahmood what do you think?

I'm guessing since it's the mobile app the URL does not make any sense at all